torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4177,6 +4189,52 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                        SkipInfo('TestUnaryUfuncs', 'test_reference_numerics_hard',
                                 device_type='cpu', dtypes=[torch.complex64])
                    )),
+    OpInfo('__radd__',
+           op=torch.Tensor.__radd__,
+           dtypes=all_types_and_complex_and(torch.bfloat16, torch.half, torch.bool),
+           sample_inputs_func=sample_inputs_rbinops,
+           supports_out=False,
+           skips=(SkipInfo('TestCommon', 'test_variant_consistency_jit',),),
+           assert_autodiffed=True,
+           autodiff_nonfusible_nodes=['aten::add'],),
```

Comment:
Add `autodiff_nonfusible_nodes` here, since they were specified before in `method_tests`. However, JIT compilation [breaks before](https://github.com/pytorch/pytorch/pull/56471/files#r616630851) it is used. So, essentially, it currently has no effect in the tests.